libraries/botframework-connector/tests/connector.test.js

```diff
@@ -44,551 +47,459 @@ const BotConnector = require('../lib');
-                var activity = createActivity();
+            it('should send a Hero card', async function () {
+                const params = createConversation();
+                const activity = createActivity();
                 activity.attachments = [
                     {
                         contentType: 'application/vnd.microsoft.card.hero',
                         content: {
                             title: 'A static image',
                             subtitle: 'JPEG image',
                             images: [
-                                { url: 'https://docs.microsoft.com/en-us/bot-framework/media/designing-bots/core/dialogs-screens.png'}
-                            ]
-                        }
-                    }
+                                {
+                                    url:
+                                        'https://docs.microsoft.com/en-us/bot-framework/media/designing-bots/core/dialogs-screens.png',
+                                },
+                            ],
+                        },
+                    },
                 ];
-				
-                client.conversations.createConversation(params)
-                    .then((result) => client.conversations.sendToConversation(result.id, activity))
-                    .then((result) => {
-                        assert(!!result.id);
-                    })
-                    .then(done, done);
+
+                const result = await client.conversations.createConversation(params);
+                const result2 = await client.conversations.sendToConversation(result.id, activity);
+                assert(!!result2.id);
             });
         });
-		
-        describe('GetActivityMembers', function() {
-            it('should have the userId', function(done) {
-                var params = createConversation();
+
+        describe('GetActivityMembers', function () {
+            it('should have the userId', async function () {
+                const params = createConversation();
                 params.activity = createActivity();
-				
-                client.conversations.createConversation(params)
-                    .then((result) => client.conversations.getActivityMembers(result.id, result.activityId))
-                    .then((result) => {
-                        result.should.matchAny((member) => member.id === user.id);
-                    })
-                    .then(done, done);
+
+                const result = await client.conversations.createConversation(params);
+                const members = await client.conversations.getActivityMembers(result.id, result.activityId);
+                members.should.matchAny((member) => member.id === user.id);
             });
-            it('should fail with invalid conversatoinId', function(done) {
-                var params = createConversation();
+            it('should fail with invalid conversationId', async function () {
```

Comment:
Note: Renaming this test required this file to be renamed, below: `Bot_Framework_Connector_SDK_Conversations_GetActivityMembers_should_fail_with_invalid_conversationId.nock.js`